cms/templates/settings.html

```diff
@@ -45,7 +47,7 @@
   <header class="mast has-subtitle">
     <h1 class="page-header">
       <small class="subtitle">${_("Settings")}</small>
-      <span class="sr">&gt; </span>${_("Schedule & Details")}
+      <span class="sr">> </span>${_("Schedule & Details")}
```

Comment:
The `&gt;` is not in the expression, it is in straight HTML, so it needs to stay as it was.